src/modules/users/infra/http/controllers/FindUserByEmail.controller.ts

```diff
@@ -0,0 +1,19 @@
+import 'reflect-metadata';
```

Comment:
Vi em alguns arquivos o import do reflect-metadata, se você fizer ele direto no app.ts ou server.ts ele funciona pra tudo que for "lido" depois, assim economiza no import